src/CTA.WebForms2Blazor/ControlConverters/HyperLinkControlConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using HtmlAgilityPack;
+
+namespace CTA.WebForms2Blazor.ControlConverters
+{
+    public class HyperLinkControlConverter : ControlConverter
+    {
+        private Dictionary<String, String> _hyperLinkControlsMap = new Dictionary<string, string>() 
+            {["navigateurl"] = "href", ["cssclass"] = "class"};
+        private readonly string _name = "a";
```

Comment:
You may want to make this a const as it shouldn't be modified, even in the constructor